dirty_cat/gamma_poisson_factorization.py

```diff
@@ -0,0 +1,478 @@
+import numpy as np
+
+import pandas as pd
```

Comment:
if you need pandas add it to the `install_requires` list in setup.py to fix the CI error